styles/stylesheet.css

```diff
@@ -0,0 +1,566 @@
+  max-width: 80%;
+}
+
+input {
+  margin: 0.5rem 0rem 1.5rem 0rem;
+  padding: 0.3rem;
+  width: 65%;
+  display: block;
+}
+
+select {
+  margin: 0.5rem 0rem 1.5rem 0rem;
+  padding: 0.3rem;
+  width: 30%;
+  display: block;
+}
+
+fieldset {
+  margin: 0.5rem 0rem 1.5rem -0.9rem;
+  border: none;
+  display: block;
+}
+
+label, legend {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 0.7em;
+  color: #103861;
+}
+
+input[type=radio] {
+  position: relative;
+  cursor: pointer;
+  height: 0.8rem;
+  width: 0.8rem;
+  margin: 0rem 1rem -0.5rem 0.5rem;
+  display: inline;
+}
+
+input[type=submit] {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  margin: 3rem;
+  padding: 0.3rem 1rem 0.3rem 1rem;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  width: 10%;
+  color: #fff;
+}
+
+input[type=submit]:hover {
+  background-color: #6a872c;
+}
+
+input[type=submit]:active {
+  background-color: #103861;
+}
+
+.form-title {
+  margin-left: 3rem;
+}
+
+.dogs-adopted div {
+  height: 5%;
+  width: 20%;
+  font-size: 0.6em;
+  display: inline-block;
+}
+
+.dogs-adopted img {
+  float: left;
+  padding-right: 1rem;
+  margin-top: 0.8rem;
+}
+
+.dogs-adopted h5 p {
+  float: right;
+}
+
+.footer-block {
+  width: 100%;
+  max-height: 30%;
+  float: left;
+  clear: both;
+  background-color: #103861;
+  text-align: center;
+  padding: 2rem;
+  color: #fff;
+}
+
+.footer-block ul {
+  margin: 0rem;
+}
+
+.footer-block li {
+  display: inline;
+  padding-right: 0.3rem;
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: 0.8em;
+}
+
+.footer-block a:link {
+  text-decoration: none;
+  color: #fff;
+}
+
+.footer-block a:visited {
+  color: #fff;
+}
+
+.footer-block a:hover {
+  color: #9bc344;
+}
+
+.footer-block a:active {
+  color: #fff;
+}
+
+/* ----- Media Queries ----- */
+/* For mobile */
+@media screen and (max-width: 576px) {
+  .container-home {
+    width: 95%;
+    margin: 1.2rem;
+  }
+  .header {
+    height: 300px;
+    background-position: center;
+  }
+  .header-logo {
+    width: 60%;
+    position: absolute;
+    top: 15%;
+    left: 7%;
+  }
+  .header h1 {
+    font-size: 0rem;
+  }
+  .header-checkout {
+    top: 25%;
+    left: 7%;
+    width: 20%;
+  }
+  ul {
+    margin: 1.2rem;
+  }
+  .dog-profiles section {
+    width: 75%;
+  }
+  .see-more {
+    width: 95%;
+    text-align: left;
+    padding: 1.5rem 0 1.5rem 0;
+  }
+  .blog-preview {
+    border-top: 1px dotted #ccc;
+    border-left: none;
+    width: 95%;
+    padding-top: 3rem;
+    margin: 1rem;
+  }
+  .blog-preview p {
+    visibility: hidden;
+    height: 0;
+  }
+}
+
+@media screen and (min-width: 577px) and (max-width: 767px) {
+  .container-home {
+    width: 95%;
+    margin: 1rem;
+  }
+  .header-logo {
+    width: 60%;
+    position: absolute;
+    top: 15%;
+    left: 4%;
+  }
+  .header h1 {
+    font-size: 0rem;
+  }
+  .header-checkout {
+    top: 50%;
```

Comment:
I'd be hesitant to place things using height%. On some resolutions this seems to be floating over content in the middle of the page.